come-lavoriamo/consuntivazione.md

```diff
@@ -1,62 +1,89 @@
+degli obiettivi in termini di ore da svolgere e cerchiamo sempre di dare la precedenza alla qualità
+del lavoro. Lasciamo che il tracker sia un semplice strumento per la consuntivazione delle ore, non
+un misuratore della nostra produttività.
+
+Tuttavia, dobbiamo essere consapevoli delle sfide che il time tracking porta con sé.
+
+Una è sicuramente quella di definire quali sono i task da attribuire ad un progetto. Non c'è
+un'interpretazione univoca nel nostro settore. Tuttavia in linea con quelli che sono [i nostri valori](https://github.com/nebulab/playbook/blob/master/su-di-noi/valori.md)
+adottiamo la seguente soluzione. Sono, quindi, da includere:
+
+- tempo di prestazione diretta o implementazione;
+- tempo di lavoro in pairing sullo stesso progetto;
+- tempo di comunicazione direttamente correlato con il lavoro che si sta svolgendo;
+- tempo di studio direttamente ed esclusivamente correlato con il lavoro che si sta svolgendo.
+
+Si dovrebbe invece switchare il tracker su un altro progetto per:
+
+- tempo di prestazione diretta o implementazione su altri progetti;
+- tempo di lavoro in pairing con un'altra persona su altri progetti;
+- tempo di comunicazione non correlato con il lavoro che si sta svolgendo;
+- tempo di studio non direttamente e non esclusivamente correlato con il lavoro che si sta svolgendo.
+
+> Un caso particolare: I tempi di attesa delle risposte del cliente che passo girandomi i pollici
 devono essere inclusi? La risposta è: non trovarsi mai nella situazione di doversi girare i pollici
 e trovare qualcosa da fare comunque. Se proprio non si ha nulla da fare questo tempo NON va incluso
 nel "tempo effettivamente svolto" ma ne va data pronta comunicazione al cliente, segnalando
 l'inefficienza.
 
-Per segnare su Float i tempi di pausa e i tempi di attesa utilizziamo il progetto `Time-Wasting`.
+Non serve far partire il tracker per i tempi di pausa ma lasciamo la libertà, per chi volesse, di
+segnare questi momenti con il progetto `Time wasting`.
+
+Alcune difficoltà potrebbero nascere soprattutto in presenza di molte attività diverse da svolgere
+contemporaneamente. La raccomandazione è quella di evitare il multitasking. Concentrarsi su task
+omogenei permette di lavorare con meno stress, più produttività e rende molto più facile l'utilizzo
+del tracker.
+
+In generale, se trovate in difficoltà nell'utilizzo del tracker rivolgetevi al vostro mentore.
+
+### Ore di permesso o di straordinario
+
+Cliccando sul tab [Presenze](https://secure.dipendentincloud.it/app/timesheet) e in seguito sul
+giorno o trascinando il puntataore su più giorni, sarà possibile inserire tutte le tipologie di
+giustificativi: [varie tipologie di Permessi (utilizza il default `Permessi`); Ferie; Malattia;
+Straordinari.](https://github.com/nebulab/playbook/blob/master/procedure/ferie-permessi-malattia.md)
+
+La scelta della tipologia, disponibile nel menù a tendina, dovrà seguire i seguenti criteri:
```

Comment:
This part belongs in "Ferie, permessi e malattia" I think. It's just more likely that people will go and look for it there, and we also have the link in case they go here instead.